### DISCUSSION TITLE
Feature Suggestion: Info Page

### DISCUSSION THREAD

**Discussion by Drac1652** [Miruro-no-kuon/Miruro#202] [Show and tell]
I love the interface and everything but there's no watch page, if you select an animeit goes directly to watching episode 1. Some animes like One piece hava 1000+ episodes, I'll need to select one episode like 700. I'll need to select it on watch page rather than selecting it while episode 1 is playing.
If you find this ideal to you, you can add watch page

**Comment by ghost** [comment]
<details><summary>I'm gessing you're referring to an anime info page</summary>
<p>

> Hey @Drac1652, I'm gessing you're referring to an anime info page that precedes the actual watch page, where users can select specific episodes before diving into the viewing experience. I like your idea of having a dedicated page for easier navigation, especially for series with a large number of episodes like "One Piece."

</p>
</details> 

<details><summary>Balancing this with Miruro's current design philosophy</summary>
<p>

> However, balancing this with Miruro's current design philosophy, which emphasizes speed and direct access to content, is a challenge. The core concept of getting viewers to the video content as quickly as possible might conflict with the introduction of an additional step. It's essential to consider how to integrate this new page without compromising the swift and seamless user experience that's central to Miruro's approach.

</p>
</details> 

Absolutely appreciate your input here! Finding that sweet spot where we can blend your suggestion with Miruro's quick-access vibe is key. How about we introduce a user preference setting that allows viewers to choose whether they land on the info page or go straight to watching the episode? This way, everyone can tailor their browsing to what suits them best, whether they prefer getting all the details upfront or jumping right into the action.

Another idea could be implementing a "quick toggle" feature on the homepage or even directly in the video player. This toggle could allow users to switch between "info mode" and "watch mode" effortlessly, without needing to dive into settings every time they want a different experience. It could be a simple switch that temporarily adjusts the navigation based on what the user feels like in that moment—more details or immediate viewing. What do you think of these approaches?

**Comment by Drac1652** [comment]
That sounds great!
And i believe others users will be happy.. Thanks for the response
